lib/torque/provider/tilejson.js

```diff
@@ -48,10 +48,39 @@
      *   Index: Array index to the properties
      * }
      */
-    proccessTile: function(rows, coord, zoom) {
+    createProccessTileWorker:function(){
+      var workerFunction = "var proccessTile ="+ this.proccessTileSerial.toString()
+      var wrapper = "; self.onmessage = function(e){var data = JSON.parse(e.data); JSON.stringify(self.postMessage(proccessTile(data.rows,data.coord, data.zoom, data.options)))}"
+      var script = workerFunction + wrapper;
+      var blob = new Blob([script], {type: "text/javascript"})
+      var worker  = new Worker(window.URL.createObjectURL(blob))
+      return worker
+    },
+    proccessTile:function(rows,coord,zoom,callback){
+      if(typeof(Worker) === "undefined"){
+        callback(this.proccessTileSerial(rows,coord,zoom, this.options))
+      }
+      else{
+        var worker = this.createProccessTileWorker()
+        worker.onmessage = function(e){
```

Comment:
I don't understand this, why the worker is terminated when recieves a message (maybe is because I don't actully know the webworkers api)